src/ClientRuntime/Microsoft.Rest.ClientRuntime.Azure.Authentication/ActiveDirectoryServiceSettings.cs

```diff
@@ -16,7 +16,7 @@ public sealed class ActiveDirectoryServiceSettings
 
         private static readonly ActiveDirectoryServiceSettings AzureSettings = new ActiveDirectoryServiceSettings
         {
-            AuthenticationEndpoint= new Uri("https://login.windows.net/"), 
+            AuthenticationEndpoint= new Uri("https://login.microsoftonline.com/"), 
```

Comment:
@markcowl Could you please explain why the "old" endpoint is preferred? I thought everything that depends on Azure Resource Manager has to go through the new endpoint (login.microsoftonline.com).